### DISCUSSION TITLE
Dependencies nightmares ended attempts to test

### DISCUSSION THREAD

**Discussion by yet-another-steve** [afadil/wealthfolio#182] [General]
I'd have loved to test your lovely looking software on my Ubuntu 20.04.5 LTS OS, but after 2 hours trying to solve dependency nightmares I will just look else where.  Maybe you could try and remember that real world users don't have the latest versions (an advantage on a LTS release is that its supported for years.)  The dependency on libwebkit2gtk-4.1-dev and all its underlying dependencies finally killed it for me.  Off for a beer to drown my sorrows ...

**Comment by afadil** [comment]
You can use devcontainer for dev mode https://github.com/afadil/wealthfolio?tab=readme-ov-file#development-with-devcontainer

you can also now self-host the app in docker
